docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -354,21 +358,22 @@ public class OidcDiscoveryRequestCustomizer implements OidcRequestFilter {
     }
 }
 ----
-<1> Restrict this filter to requests targeting the OIDC discovery endpoint only.
+<1> Restrict this filter to request targeting of the OIDC discovery endpoint only.
```

Comment:
```suggestion
```

Reply:
IMHO it does not quite work

Reply:
Thanks @sberyozkin. Just to confirm, is your suggestion to remove sentence 361 entirely?

Reply:
@sheilamjones Please remove it

Reply:
Done